packages/demo/webpack.config.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
         new CopyPlugin({
             patterns: [
                 path.resolve(__dirname, 'node_modules', 'jquery', 'dist', 'jquery.slim.min.js'),
-                path.resolve(__dirname, '..', '..', 'node_modules', 'chosen-js', 'chosen.jquery.min.js'),
+                path.resolve(__dirname, 'node_modules', 'react-vapor', 'node_modules', 'chosen-js', 'chosen.jquery.min.js'),

```

Comment:
We have 2 `node_modules` in this line, is it wanted? :thinking:

Reply:
Yes, [pnpm doesn't use a flat node_modules structure](https://pnpm.io/5.x/symlinked-node-modules-structure)